scripts/lib/mkcloud-libvirt.sh

```diff
@@ -57,36 +57,85 @@ function libvirt_net_start()
         ip link set mtu 9000 dev $dev
     done
 
+    boot_mkcloud=/etc/init.d/boot.mkcloud
+    boot_mkcloud_d="$boot_mkcloud.d"
+    boot_mkcloud_d_cloud="$boot_mkcloud_d/$cloud"
+
     if [ -z "$NOSETUPPORTFORWARDING" ] ; then
+        # FIXME: hardcoded assumptions about admin net host range
         nodehostips=$(seq -s ' ' 81 $((80 + $nodenumber)))
```

Comment:
I was in a hurry while reviewing this. What I understand from this ``nodehostips=$(seq -s ' ' 81 $((80 + $nodenumber)))`` is that the nodehostips is actually all the nodes in the cluster. We just need to forward the ports for `Crowbar Admin`, `Controller` and `Pacemaker HAWK Nodes` related endpoints. But it gets complicated as we would then have to talk to Crowbar, but if we have a simpler way to do the same it would be great to consider this.